docs/kb/api-version-4.md

```diff
@@ -1,48 +1,98 @@
-Failure to do so will result in an error about the API being requested.
+# Transitioning from version 3 to version 4
 
-The biggest change in the APIs in this version are that we now allow for multiple prospect records with the same email address.
-A few examples of what that means:
+If you are looking to upgrade your account to 
+[allow multiple prospects with the same email address](http://help.pardot.com/customer/portal/articles/2461386-how-can-i-find-out-if-my-account-allows-multiple-prospects-with-the-same-email-address-),
+and you use the api, you will need to follow this guide.
 
-* Prospect Create by email will create a new prospect for each time the call is made.
-* Prospect Upsert by email will also always create under this specification.
-* Prospect Update by email will no longer work (we don't necessarily know which prospect you're talking about)
+## Determining when these changes are applicable
 
-### Requesting an API key
-
-An exception to this rule is requesting an `api_key` through the login API.
-You may use `/version/3` or `/version/4` for this call.
-To let you know that you must use version 4, we will return a `<version>4</version>` tag in the response.
-This version tag will not be included if your account cannot use version 4.
+When your account has access to version 4, your API session will be cleared and you must log in again.
+Upon logging in, a new field will be returned in the login response. 
+This will be your indication that you may no longer use version 3 (or any other previous version), and must now use the version 4 syntax.
+Here's what that will look like:
 
 Request:
 ```
 POST: https://pi.pardot.com/api/login/version/3
 message body: email=<email>&password=<password>&user_key=<user_key>
 ```
 
-Response:
+Response (Before transition):
+```
+<rsp stat="ok" version="1.0">
+  <api_key>1234abcd</api_key>
+</rsp>
+```
+
+Response (After transition):
 ```
 <rsp stat="ok" version="1.0">
   <api_key>1234abcd</api_key>
   <version>4</version>
 </rsp>
-```
\ No newline at end of file
+```
+
+*Note: Using version 3 to login will continue to work, even though all other endpoints will require version 4*
+
+## Request Path Changes
+
+You will need to modify your request to use the `/version/4` path. For example:
+
+Before transition:
+```
+GET: https://pi.pardot.com/api/prospect/version/3/read/id/1
+```
+After transition:
+```
+GET: https://pi.pardot.com/api/prospect/version/4/read/id/1
+```
+
+
+## Prospect API changes
+
+Most of the changes in version 4 occur around the prospect APIs. 
+When you begin using the version 4 APIs, your logic should know about these changes ([full documentation can be found here](/kb/api-version-4/prospects))
+
+* **New reference field:**  Prospects can now be referenced using the Salesforce CRM Identifier, known to the api as the `fid` field.
+This `fid` field refers to the Salesforce identifier applied to Leads or Contacts. 
```

Comment:
<img width="859" alt="screen shot 2016-08-24 at 7 43 14 am" src="https://cloud.githubusercontent.com/assets/2990057/17929417/734ed7c8-69ce-11e6-95e6-d6ed83de7f3f.png">
How this now looks (since it probably won't look right in the preview)